ansible/library/announce_routes.py

```diff
@@ -50,13 +50,14 @@
 
 
 def get_topo_type(topo_name):
-    pattern = re.compile(r'^(t0|t1|ptf|fullmesh|dualtor|t2)')
+    pattern = re.compile(r'^(t0|t1|ptf|fullmesh|dualtor|t2|mgmttor)')
     match = pattern.match(topo_name)
     if not match:
-        raise Exception("Unsupported testbed type - {}".format(topo_name))
+        logger.warning("Unsupported testbed type - {}".format(topo_name))
+        return "unsupported"
     topo_type = match.group()
-    if 'dualtor' in topo_type:
-        # set dualtor topology type to 't0' to avoid adding it in each test script.
+    if topo_type in ['dualtor', 'mgmttor']:
```

Comment:
dualtor in topo_type accomodates all dualtor testbeds, please take the same change as in #3553

Reply:
topo_type will be exact `dualtor` no matter input topo_name is `dualtor` or `dualtor-56`

Reply:
I see. you are right.